components/Loading.vue

```diff
@@ -0,0 +1,186 @@
+<template id="loading-component">
+  <div class="background">
+    <div class="fire">
+      <div class="flames">
+        <div class="flame" />
+        <div class="flame" />
+        <div class="flame" />
+        <div class="flame" />
+      </div>
+      <div class="logs" />
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Loading'
+}
+</script>
+
+<style lang="scss" scoped>
+$dark-blue: #061A2C;
+$brownl: #70392F;
+$brownd: #612E25;
+$yellow: #FFDC01;
+$orange: #FDAC01;
+$red: #F73B01;
+$animationtime: 1.5s;
+
+.background {
+  position: fixed;
+  width: 100vw;
+  height: 100vh;
+  background-color: $dark-blue;
+  z-index: 1000000;
+
+  .fire {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -25%);
+    height: 20vw;
+    width: 20vw;
+    //background-color:$red;
```

Comment:
do we still need these comments?

Reply:
I could remove them if you want, it was just from codepen LOL